pom.xml

```diff
@@ -217,6 +231,11 @@
         <version>2.6</version>
       </dependency>
       <dependency>
+        <groupId>commons-io</groupId>
+        <artifactId>commons-io</artifactId>
+        <version>2.5</version>
```

Comment:
@benzonico I mean there are not so many uses in main code, i.e. you can still preserve it for tests, while remove from packaging in plugin.